_data/icons/snx.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "https://ipfs.synthetix.io/ipfs/QmQPHTTpgSjvP18Yc522ZxHYKRwLLxeR6xWdEdcw1XHwZo",
```

Comment:
the current format url is wrong, it should be starting with ipfs://

Reply:
My Address

Reply:
[]()__